libs/CHNcalendar/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "CHNcalendar",
+  "filename": "CHNcalendar.js",
+  "version": "1.0",
+  "description": "农历日历",
+  "homepage": "http://gomebuy.gome.com.cn/",
+  "keywords": [],
+  "maintainers": [
+    {
+      "name": "grayhanabi",
+      "web": "https://github.com/grayhanabi",
+      "mail": "grayhanabi@gmail.com"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/staticfile/static"
```

Comment:
I can't find this repo, does it opensourced ? We do not accept non opensource code.